modules/ssh/ssh.go

```diff
@@ -30,6 +33,14 @@ type Host struct {
 	OverrideSshAgent *SshAgent // enable an in process `SshAgent` for connections to this host (disabled by default)
 }
 
+type ScpDownloadOptions struct {
+	FileNameFilter string //bash style match string for files
+	MaxFileSizeMB  int    //Don't grab any files > MaxFileSizeMB
```

Comment:
`MaxNumFiles` too?

Reply:
My main thought is to avoid downloading everything on the entire computer if you accidentally specify `/` as a path. If `MaxNumFiles` is exceeded, you stop downloading further files, and add that as a message in the MultiError. 

BTW, does this code only download files or does it recursively download folders too? Either way, it should be documented.

Reply:
This code will only download files within the folder you specified. It won't recursively download subfolders or follow symlinks.

So If you accidentally specify `/` it won't just download your whole computer :-)

Updated comments on the method to say the following:
```
// ScpFileFromE downloads all the files from remotePath on the given host using SCP 
// and returns an error if the process fails. NOTE: only files within remotePath will
// be downloaded. This function will not recursively download subdirectories or follow
// symlinks.
```